### PR TITLE
chore(kommander): bump UI to v2.25.2, KCL to v0.4.5

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: stable/kommander/Chart.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ maintainers:
 - name: alejandroEsc
 - name: jimmidyson
 name: kommander
-version: 0.4.4
+version: 0.4.5
```
Comment (juliangieseke) [inline stable/kommander/Chart.yaml]: i dont care too much, but should this be tied to KCL version? or is it coincidence?

Comment (jimmidyson) [reply stable/kommander/Chart.yaml]: It's actually just coincedence... honest!

**Comment by makkes** [comment]
We're facing TeamCity issues: https://mesosphere.slack.com/archives/CQ7J6EWG3/p1581674571098100

**Comment by juliangieseke** [comment]
yeah, going to merge this. its a simple change and will be deployed on testing cluster right away. charts arent used anywhere directly, so risk of breaking anything is _very_ low.

**Pull request merged**